yahoo_fantasy_api/league.py

```diff
@@ -121,13 +121,13 @@ def teams(self):
             teams[key] = team
         return teams
 
-    def matchups(self):
-        """Retrieve matchups data for current week
+    def matchups(self, week=None):
+        """Retrieve matchups data for a given week. Defaults to current week.
 
         :return: Matchup details as key/value pairs
         :rtype: dict
```

Comment:
We should include the new parameter you added here.  This gets picked up by the documentation.